scripts/fork_fuzz_bots.py

```diff
@@ -25,6 +25,7 @@
 # with the value as the whale address.
 # Note that if a token is missing in this mapping, we will try to
 # call `mint` on the trading token to fund.
+# TODO we need a better method to ensure this list of whales doesn't get stale
```

Comment:
would be an easy test to add to CI
fund a bot from the whale, ensure success for some X base